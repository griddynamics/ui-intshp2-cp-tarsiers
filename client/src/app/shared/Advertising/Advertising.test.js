import React from 'react';
import { shallow } from 'enzyme';
import AdvertisingArea from './index';

xdescribe('<Advertising />', () => {
  it('should match its snapshot', () => {
    const wrapper = shallow(<AdvertisingArea />);

    expect(wrapper).toMatchSnapshot();
  });

  const props = { htmlSnippet: '' };

  it('should renders without crashing', () => {
    expect(shallow(<index {...props} />).length).toEqual(1);
  });
});
