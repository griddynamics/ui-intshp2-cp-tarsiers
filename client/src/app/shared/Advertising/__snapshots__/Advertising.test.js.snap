// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Advertising /> should match its snapshot 1`] = `
<section
  className="advert"
>
  <Dynamic
    html=""
  />
</section>
`;
